src/components/Avatar/Avatar.component.tsx

```diff
@@ -119,13 +138,21 @@ export interface AvatarProps extends LightingProps, EnvironmentProps, Omit<BaseM
    * Bloom post-processing effect.
    */
   bloom?: BloomConfiguration;
+  /**
+   * Spawn effect on mount.
+   */
+  onMountEffect?: SpawnState['onMountEffect'];
+  /**
+   * Spawn animation on mount.
+   */
+  onMountAnimation?: SpawnState['onMountAnimation'];
 }
 
 /**
  * Interactive avatar presentation with zooming and horizontal rotation controls.
  * Optimised for full-body and half-body avatars.
  */
-export const Avatar: FC<AvatarProps> = ({
+const Avatar: FC<AvatarProps> = ({
```

Comment:
I'm not sure how the library still works after removing this explicit export.

Check `src/library.ts` -> this file explicitly states which components are built into Visage node module.

Looks like in the built package the `AvatarWrapper` is omitted so state won't work.

Reply:
the hacks comes in the index were we export the Avatar as the Wrapper 

`import Avatar, { CAMERA } from './Avatar.component';

export { Avatar, CAMERA };
`